exotic/api/filters.py

```diff
@@ -1,34 +1,73 @@
+        "MObs CV": {"name": "CV", "desc": "MObs CV", "fwhm": ("350.0", "850.0")},
+
+        # Astrodon CBB; Source(s): George Silvis; https://astrodon.com/products/astrodon-exo-planet-filter/
+        "Astrodon ExoPlanet-BB": {"name": "CBB", "desc": "Astrodon ExoPlanet-BB", "fwhm": ("500.0", "1000.0")},
+
+        # LCO; Source(s): Kalee Tock and Michael Fitzgerald; 0.4m & 1.0m Telescopes; https://lco.global/observatory/instruments/filters/
+        "LCO Bessell U": {"name": "N/A", "desc": "LCO Bessell U", "fwhm": ("325.0", "375.0")},
+        "LCO Bessell B": {"name": "N/A", "desc": "LCO Bessell B", "fwhm": ("391.6", "480.6")},
+        "LCO Bessell V": {"name": "N/A", "desc": "LCO Bessell V", "fwhm": ("502.8", "586.8")},
+        "LCO Bessell R": {"name": "N/A", "desc": "LCO Bessell R", "fwhm": ("561.7", "719.7")},
+        "LCO Bessell I": {"name": "N/A", "desc": "LCO Bessell I", "fwhm": ("721.0", "875.0")},
 
-        # MObs Clear Filter, Source: Martin Fowler
-        ('MObs CV', 'CV'): (350.00, 850.00),
+        "LCO Pan-STARRS w": {"name": "N/A", "desc": "LCO Pan-STARRS w", "fwhm": ("404.2", "845.8")},
+        "LCO Pan-STARRS Y": {"name": "N/A", "desc": "LCO Pan-STARRS Y", "fwhm": ("948.0", "1060.0")},
+        "LCO Pan-STARRS zs": {"name": "N/A", "desc": "LCO Pan-STARRS zs", "fwhm": ("818.0", "922.0")},
 
-        # Astrodon CBB: George Silvis: https://astrodon.com/products/astrodon-exo-planet-filter/
-        ('Astrodon ExoPlanet-BB', 'CBB'): (500.00, 1000.00),
+        "LCO SDSS u'": {"name": "N/A", "desc": "LCO SDSS u'", "fwhm": ("325.5", "382.5")},
+        "LCO SDSS g'": {"name": "N/A", "desc": "LCO SDSS g'", "fwhm": ("402.0", "552.0")},
+        "LCO SDSS r'": {"name": "N/A", "desc": "LCO SDSS r'", "fwhm": ("552.0", "691.0")},
+        "LCO SDSS i'": {"name": "N/A", "desc": "LCO SDSS i'", "fwhm": ("690.0", "819.0")},
+}
 
-        # LCO, Source: Kalee Tock & Michael Fitzgerald, https://lco.global/observatory/instruments/filters/
-        ('LCO Bessell B', 'N/A'): (391.60, 480.60), ('LCO Bessell V', 'N/A'): (502.80, 586.80),
-        ('LCO Bessell U', 'N/A'): (325.0, 375.00), ('Harris B', 'N/A'): (377.75, 484.65),
-        ('LCO Pan-STARRS w', 'N/A'): (404.20, 845.80), ('LCO Pan-STARRS w', 'N/A'): (404.20, 845.80),
-        ('LCO Pan-STARRS zs', 'N/A'): (818.00, 922.00), ('LCO SDSS g\'', 'N/A'): (402.00, 552.00),
-        ('LCO SDSS r\'', 'N/A'): (552.00, 691.00), ('LCO SDSS i\'', 'N/A'): (690.00, 819.00)}
\ No newline at end of file
+# aliases to back-reference naming standard updates
+fwhm_alias = {
+              "J NIR 1.2 micron": "Near-Infrared J",
+              "J NIR 1.2micron": "Near-Infrared J",
+              "H NIR 1.6 micron": "Near-Infrared H",
+              "H NIR 1.6micron": "Near-Infrared H",
+              "K NIR 2.2 micron": "Near-Infrared K",
+              "K NIR 2.2micron": "Near-Infrared K",
+              "MObs CV": "Clear (unfiltered) reduced to V sequence",
```

Comment:
I think that's backwards. The intent of alias is to reference 'undefined_name' to defined name. MObs CV is defined above already.

Reply:
Changes resolved this.